backend/README.md

```diff
@@ -0,0 +1,33 @@
+# Backend Server TL;DR
+
+This folder contains code to run the server for TL;DR
+
+## Installation
+
+1. Clone the repository
+
+2. Check that you have [node](https://nodejs.org/en/) and [npm](https://www.npmjs.com/) installed 9
+
+    - to check you have node and installed:
+        - `node -v`
+        - `npm -v`
+
+3. make sure you are in the `backend/` directory
+
+4. Run `npm install` to install all relevant packages
+
+## Running the Server
+
+1. make sure you are in the `backend` directory
+
+2. Run `node backend_test.js` to run the server
```

Comment:
small nit: should we use a better name for the backend entry?

Reply:
agreed

Reply:
For sure, this was just a placeholder